F1WM.UnitTests/Services/NewsServiceTests.cs

```diff
@@ -167,5 +167,19 @@ public async Task ShouldGetRelatedNews()
 			newsRepositoryMock.Verify(r => r.GetRelatedNews(id, now, count), Times.Once);
 		}
 
+		[Fact]
+		public async Task ShouldGetSearchResults()
+		{
+			var term = "maldonado";
+			var news = fixture.Create<PagedResult<NewsSummary>>();
+			var now = new DateTime(2017, 9, 19);
+
+			timeServiceMock.SetupGet(t => t.Now).Returns(now);
```

Comment:
This setup is redundant. You call `service.SearchNews(term, 1, 20, now);`, so that it takes the date you provided via method parameter, not via TimeService. It's actually two separate cases to test (with and without `before` parameter), but more important one is to check that it works correctly with `before = null`, so please change the call in line 180 to `service.SearchNews(term, 1, 20, null);`